ui/lib/core/addon/templates/components/form-field.hbs

```diff
@@ -229,7 +226,27 @@
           theme=(or attr.options.theme 'hashi')
         }}
         @helpText={{attr.options.helpText}}
-      />
+      >
+      <button>{{if attr.options.allowReset "yes" "no"}}</button>
```

Comment:
What's the purpose of this button?

Reply:
whooops leftover from debugging tests 😂 🤦